src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Card, CardContent, CardHeader, Grid, Typography
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  bold: {
+    fontWeight: 'bold'
+  }
+});
+
+const HealthCardDisplay = ({ patientData }) => {
+  const classes = useStyles();
+
+  const immunizationCode = (codings) => {
+    if (codings.length === 0) return '';
+
+    const coding = codings[0];
+    return coding.system ? `${coding.system}#${coding.code}` : coding.code;
```

Comment:
Sometimes the codesystem contains an entire HL7 URL (the SHCs from our reference implementation have this). Do we want to truncate the URL from the system if it exists?

For example, a code from our reference implementation displays as  **`http://hl7.org/fhir/sid/cvx#212`**, which seems kind of verbose.

Reply:
We have a task to use more human readable names so this will probably get changed soon, probably not worth touching for now.

Reply:
Good point, I forgot that task was in our current sprint. Makes sense to keep as is for now.